### PR TITLE
Add non-failing lints.

### PR DESCRIPTION
**Pull request by mariusvniekerk**
Requires conda-forge/conda-smithy#778

### CODE REVIEW

**Comment by mariusvniekerk** [comment]
This is expected to fail until some variant of the smithy pr lands

**Comment by jakirkham** [comment]
LGTM

**Comment by jakirkham** [comment]
Going to go ahead with this as I'm a little worried the linter won't handle the new `conda-smithy` well without this.

**Comment by jakirkham** [comment]
That said, if there are thoughts/issues, please feel free to raise them.

File: conda_forge_webservices/linting.py [hunk 1]
```diff
@@ -117,6 +119,10 @@ def compute_lint_message(repo_owner, repo_name, pr_id, ignore_base=False):
                 all_pass = False
                 messages.append("\nFor **{}**:\n\n{}".format(rel_path,
                                                              '\n'.join(' * {}'.format(lint) for lint in lints)))
+            if hints:
+                messages.append("\nFor **{}**:\n\n{}".format(rel_path,
+                                                             '\n'.join(' * {}'.format(lint) for lint in lints)))
```
Comment (isuruf) [inline conda_forge_webservices/linting.py]: Should be `format(hint) for hint in hints`

**Pull request merged**